pkg/util/util.go

```diff
@@ -122,3 +123,19 @@ func GenFileUrl(location string, os string) string {
 	}
 	return "file://" + urlPath
 }
+
+// ConvertKeyValueStringToMap converts String Slice of Parameters to a Map[String]string
+// Each value of the slice is expected to be in the key=value format
+// Values that do not conform to this "spec", will be ignored
+func ConvertKeyValueStringToMap(params []string) map[string]string {
+	result := make(map[string]string, len(params))
+	for _, param := range params {
+		str := strings.Split(param, "=")
+		if len(str) != 2 {
+			glog.Warningf("Parameter %s is not in the expected key=value format", param)
```

Comment:
This should fatal out if it's not in the expected format rather than warn.

Reply:
I was actually on the fence about this. Y'all are the experts on CLIs, so I'll take your word for it and fix it tomorrow :)